src/box/applier.h

```diff
@@ -194,4 +190,19 @@ applier_resume(struct applier *applier);
 void
 applier_pause(struct applier *applier);
 
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+/**
+ * Return string, which represent remote URI
+ * for this @a applier.
```

Comment:
IMO it's worth mentioning in the comment that this function uses `tt_static_buf` for storing the result.

Reply:
fixed